docs/resources/cost_profile.md

```diff
@@ -87,9 +88,11 @@ resource "rafay_cost_profile" "tfdemocostprofile-onprem" {
     provider_type = "onprem"
     installation_params {
       on_prem {
-        custom_pricing {
-          cloud_credentials_name = "sample"
-        }
+        cpu = "2"
```

Comment:
Here in examples can we have some lower decimal values?
 Check here -- https://github.com/opencost/opencost/blob/develop/pkg/cloud/providerconfig.go#L218